xarray/core/dataarray.py

```diff
@@ -3505,6 +3505,202 @@ def pad(
         )
         return self._from_temp_dataset(ds)
 
+    def idxmin(
+        self,
+        dim: Hashable = None,
+        skipna: bool = None,
+        fill_value: Any = np.NaN,
+        keep_attrs: Optional[bool] = False,
```

Comment:
@toddrjen I should have spotted this earlier, and I appreciate you've gone through a lot of iteration on this PR already, thank you.

Could we conform to the global config on `keep_attrs`? It should only be setting this default to `None` and adding a couple of extra lines of code, copied from any (I hope...) of the other methods that take `keep_attrs`. Let me know any issues at all.

Reply:
I will just delegate this to `argmin` and `argmax` and use whatever attributes they come up with.